aiosmtpd/smtp.py

```diff
@@ -19,12 +19,13 @@
     List,
     NamedTuple,
     Optional,
+    Tuple,
     Union,
 )
 from warnings import warn
 
 
-__version__ = '1.2.2'
+__version__ = '1.2.2+'
```

Comment:
Unrelated to this PR specifically, but we should update the build/release helper to add the + after the release is tagged/built/pushed

Reply:
I believe '+' is not allowed in *canonical* version: https://www.python.org/dev/peps/pep-0440/#public-version-identifiers
`1.2.3a0` sounds better :)

Reply:
Great point! Even better!

Reply:
Awesome suggestion! I was actually just trying out `[skip ci]` there, but didn't have any idea what to change, so added a "+" there.

I'll change that to "a0" on my next commit.